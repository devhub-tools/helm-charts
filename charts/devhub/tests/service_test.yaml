suite: Service
templates:
  - service.yaml
release:
  name: devhub
  namespace: devhub

tests:
  - it: should create service by default
    set:
      service:
        annotations:
          custom-annotation: "custom-value"
    asserts:
      - matchSnapshot: {}
