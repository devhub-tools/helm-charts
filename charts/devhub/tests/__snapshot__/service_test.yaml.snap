should create service by default:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        custom-annotation: custom-value
      labels:
        app.kubernetes.io/instance: devhub
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: devhub
        app.kubernetes.io/version: v1.5.1
        helm.sh/chart: devhub-2.1.0
      name: devhub
    spec:
      ports:
        - name: http
          port: 80
          protocol: TCP
          targetPort: http
        - name: postgres-proxy
          port: 5432
          protocol: TCP
          targetPort: postgres-proxy
      selector:
        app.kubernetes.io/instance: devhub
        app.kubernetes.io/name: devhub
      type: ClusterIP
